src/contexts.ts

```diff
@@ -1,6 +1,6 @@
 import { createContext, QRL } from '@builder.io/qwik'
 
-export const QwikqlURLContext = createContext<{ url: string }>('qwikql.url')
+export const QwikqlURLContext = createContext<{ url: string, after$?: QRL<(args: any) => void> }>('qwikql.url')
```

Comment:
As mentioned [in another comment](https://github.com/TahaSh/qwikql/pull/1/files#r1056884174), this is `any` type because I couldn't quite figure out how to import the response type from `graphql-request`